src/code42cli/cmds/profile.py

```diff
@@ -21,14 +21,18 @@ def profile():
     "-n",
     "--name",
     required=True,
-    type=str,
     help="The name of the Code42 CLI profile to use when executing this command.",
 )
 server_option = click.option(
-    "-s", "--server", required=True, type=str, help="The url and port of the Code42 server."
+    "-s", "--server", required=True, help="The url and port of the Code42 server."
 )
 username_option = click.option(
-    "-u", "--username", required=True, type=str, help="The username of the Code42 API user."
+    "-u", "--username", required=True, help="The username of the Code42 API user."
+)
+password_option = click.option(
+    "--password",
+    help="The password for the Code42 API user. Only recommended if command needs to be run without "
```

Comment:
The second sentence here kind of sounds like we are recommending not saving a password if possible. I _think_ what we're trying to say is that you can avoid using the flag so that the command doesn't end up in your bash history, so I would maybe make the second sentence something like:

```
if this option is omitted, interactive prompts will be used to obtain the password.
```

Reply:
yeah, I struggled with the right way to word this. I like your sentence better.